index.html

```diff
@@ -11,272 +11,227 @@
-            fill="white"
-            stroke="#262626"
-            stroke-width="0.496227"
-            stroke-miterlimit="10"
-          />
-          <path
-            d="M79.7207 237.489L79.7188 248.712L68.8173 257.512L68.9336 246.702L79.7207 237.489Z"
-            fill="white"
-            stroke="#262626"
-            stroke-width="0.496227"
-            stroke-miterlimit="10"
-          />
-          <path
-            d="M119.26 254.197L119.351 272.173L102.444 282.971L102.525 265.614L119.26 254.197Z"
-            fill="white"
-            stroke="#262626"
-            stroke-width="0.496227"
-            stroke-miterlimit="10"
-          />
-        </svg> -->
-      </div>
-      <section>
-        <div class="headline">
-          <h1>
-            Hey There. <br />
-            I'm Dami
-          </h1>
-          <p class="tagline">I am a Software Developer</p>
-          <p class="descr">
-            I can help you build a product, feature or website. Look through
-            some of my work and experience! If you like what you see and have a
-            project you need coded, don't hesitate to contact me.
-          </p>
+      <div class="main-wrapper">
+        <section class="full-h bkg-black" id="headline-wrapper">
+          <div class="flex-all">
+            <h1 class="crete">
+              Hey There. <br />
+              I'm Dami
+            </h1>
+            <p class="tagline">I am a Software Developer</p>
+            <p class="descr">
+              I can help you build a product, feature or website. Look through
+              some of my work and experience! If you like what you see and have
+              a project you need coded, don't hesitate to contact me.
+            </p>
+            <div class="socials">
+              <a href="https://github.com/adamilare">
+                <img src="images/git.svg" alt="git logo" class="social-svg" />
+              </a>
+              <a href="https://linkedin.com/in/adamilare">
+                <img src="images/in.svg" alt="linkedin logo" />
+              </a>
+              <a href="https://angelist.com/">
+                <img
+                  src="images/angelist.svg"
+                  alt="Angelist logo"
+                  class="social-svg"
+                />
+              </a>
+              <a href="https://twitter">
+                <img
+                  src="images/twitter.svg"
+                  alt="twitter logo"
+                  class="social-svg"
+                />
+              </a>
+              <a href="https://medium">
+                <img
+                  src="images/medium.svg"
+                  alt="medium logo"
+                  class="social-svg"
+                />
+              </a>
+            </div>
+          </div>
+        </section>
+        <section class="my-works">
+          <h1 id="works-header">My Recent Works</h1>
```

Comment:
- [ ] Kindly change this `h1` tag to another `hx` tag like h2, h3... (this applies to the `h1` tags below this one) for a better heading structure since in a webpage there should only be one `h1` which is already taken for the main heading 👍